src/main/java/com/example/qnacomunity/service/QnaService.java

```diff
@@ -203,6 +209,8 @@ public void deleteQuestion(MemberResponse memberResponse, Long questionId) {
       elasticSearchService.delete(questionId);
 
     } catch (Exception e) {
+      log.error("ES 연동 실패: {}", e.getMessage());
```

Comment:
e.message를 찍어도 좋고, exception 원인 파악자체가 목적이라면 아래처럼 stackTrace가 찍히도록 e를 넣어주셔도 좋을 것 같습니다. 
```suggestion
      log.error("ES 연동 실패", e);
```